R/utils_groupcomparison_plots.R

```diff
@@ -86,7 +122,7 @@
     legend.size, log_adjp
 ) {
     Protein = NULL
-    
+    print(input)
```

Comment:
where will this be printed?

Reply:
Commit [remove debug](https://github.com/Vitek-Lab/MSstats/pull/107/commits/1c343e331f9735080d5a9c3af4a69fd8683525f7)